### PR TITLE
Persistence TestKit documentation article

### PR DESCRIPTION
**Pull request by valdisz**
A new article which will cover new Persistence TestKit.

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
Part of #3877

**Comment by valdisz** [comment]
@Aaronontheweb do you like wording better? Can I continue in the same manner?

**Comment by Aaronontheweb** [comment]
@valdisz I'll take a look at it, since I would really love to have this up on the website to go along with the v1.4.0-beta2 release we just pushed

**Comment by valdisz** [comment]
@Aaronontheweb so... do you like it now?

File: src/core/Akka.Persistence.TestKit.Tests/Actors/CounterActor.cs [hunk 1]
```diff
@@ -69,14 +70,17 @@ public class CounterActorTests : PersistenceTestKit
         [Fact]
         public async Task CounterActor_internal_state_will_be_lost_if_underlying_persistence_store_is_not_available()
         {
-            await WithJournalWrite(write => write.Fail(), () =>
+            await WithJournalWrite(write => write.Fail(), async () =>
```
Comment (Aaronontheweb) [inline src/core/Akka.Persistence.TestKit.Tests/Actors/CounterActor.cs]: Ah, that would do it

**Comment by sean-gilliam** [comment]
Looks like you forgot to add the article to the `articles/toc.yml` so it won't show up in the website's sidebar nav.

**Comment by Aaronontheweb** [comment]
@sean-gilliam ah shit - mind sending in a PR for that?

**Comment by sean-gilliam** [comment]
@Aaronontheweb Sure thing #4001

**Pull request merged**